assets/react/controllers/Incident/CommentBox.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { Button, Input } from "antd";
+
+import store, { setEvents } from "./incidentStore";
+import { useTranslation } from "react-i18next";
+
+async function _handleCommentSubmit(id, comment) {
+  const httpClient = new window._auth.httpClient();
+  return await httpClient.post(
+    window.Routing.generate("api_incidents_add_comment_item", { id }),
+    { comment },
+  );
+}
+
+export default function () {
+  const { incident } = store.getState();
```

Comment:
same as above; should we use `useStore` instead if importing the store directly?

Reply:
same as above; this component do not need reactivity.